.github/workflows/build-test.yml

```diff
@@ -376,47 +315,29 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        jupyter: [raw, local, remote]
+        jupyter: [raw, local]
         python: [nonConda, conda]
         pythonVersion: [3.9]
-        # integration: Tests with VS Code, Python extension & real Jupyter
-        # notebook: Notebook Tests with VS Code, Python extension & real Jupyter
-        # notebookAndWebview: Tests with VS Code (possible Notebooks), WebViews, Python extension & real Jupyter
-        test-suite: [integration, notebook]
```

Comment:
All webview tests (variable webview tess) run as part of the standard notebook tests today, hence we don't need a separate job.
I.e. we don't need a `test-suite` section anymore